tests/lit-tests/darwin-minos-ver.ispc

```diff
@@ -0,0 +1,15 @@
+// The test checks that the triple contain minimum OS version if provided.
+
+// RUN: %{ispc} %s --nostdlib --target-os=macos --emit-llvm-text --nowrap --target=host -o - | FileCheck %s --check-prefix=CHECK-MACOS-DEFAULT
```

Comment:
We advertise it for macOS/iOS, but test it for macOS only. Should we do that for iOS?..

Though that might be not straightforward, as we don't enable iOS in default CI builds. We should probably test it at least locally.